spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -37,7 +37,8 @@
       page_count: [rand(1..1000).to_s],
       rights: ["http://creativecommons.org/licenses/by-nc-sa/3.0/us/"],
       visibility: InvenioRdmRecordConverter::OPEN_ACCESS,
-      related_url: ["https://doi.org/10.5438/55e5-t5c0"]
+      related_url: ["https://doi.org/10.5438/55e5-t5c0"],
+      acknowledgments: ["this is an acknowledgement"]
```

Comment:
For some reason the GenericFile class uses the spelling "acknowledgments" which is apparently the British English preferred way.

Reply:
yeah - the moment you have picky words like this, you get subtle typos ;)